pkg/backend/httpstate/backend.go

```diff
@@ -480,33 +480,41 @@ func (b *cloudBackend) Name() string {
 }
 
 func (b *cloudBackend) URL() string {
-	user, _, err := b.CurrentUser()
+	user, err := b.currentUser(context.Background())
 	if err != nil {
 		return cloudConsoleURL(b.url)
 	}
-	return cloudConsoleURL(b.url, user)
+	return cloudConsoleURL(b.url, user.Username)
 }
 
 func (b *cloudBackend) SetCurrentProject(project *workspace.Project) {
 	b.currentProject = project
 }
 
-func (b *cloudBackend) CurrentUser() (string, []string, error) {
-	return b.currentUser(context.Background())
+func (b *cloudBackend) CurrentUser() (*workspace.Account, error) {
+	account, err := b.currentUser(context.Background())
+	if err != nil {
+		return nil, err
+	}
+	return &account, nil
```

Comment:
optional: can shorten this to:

```go
return &account, err
```

It's understood that the returned value is a undefined if the error is non-nil.

Reply:
Sure